memo_app.rb

```diff
@@ -2,9 +2,48 @@
 
 require 'sinatra'
 require 'json'
+require 'pg'
+require 'dotenv'
+
+Dotenv.load
+
+# MemoClass for Run SQL
+class Memo
+  class << self
+    def connection
+      PG.connect(
+        host: ENV['DB_HOST'],
+        password: ENV['DB_PASS'],
+        user: ENV['DB_USER'],
+        dbname: ENV['DB_NAME'],
+        port: ENV['DB_PORT']
+      )
+    end
+
+    def show
+      connection.exec('SELECT * FROM MEMO ORDER BY ID')
```

Comment:
```suggestion
      connection.exec('SELECT * FROM memo ORDER BY id')
```

テーブル名やカラム名は小文字で統一した方が読みやすいと思います。（ここ以外のSQLも同様）

Reply:
小文字統一のほうがメリハリがあって読みやすいと思いました。
下記コミットで修正いたしました。
https://github.com/azitama6/memo-app/pull/2/commits/c0adc394f4a9566956842814616d2b01bef55231